### ISSUE TITLE
EncapsulateFieldRefactoring encapsulates Private fields using Public properties 

### ISSUE THREAD

**Issue by BZngr** [rubberduck-vba/Rubberduck#5421]
**Rubberduck version information**
Version 2.5.0.27138
OS: Microsoft Windows NT 10.0.18362.0, x64
Host Product: Microsoft Office x86
Host Version: 16.0.12527.20242
Host Executable: EXCEL.EXE

**Description**
The title says it all.

**Expected behavior**
Encapsulation Properties should provide the same `Accessibility` to the field as it had before encapsulation.

**Comment by SmileyFtW** [comment]
The title really doesn't explain the issue IMNSHO. The bug report form is formatted the way it is so that others may read and understand the issue - learning something in the process.
Would greatly encourage everyone to follow the report format.
Respectfully,
Smiley FtW

**Comment by retailcoder** [comment]
Yeah looking at the diff makes it more obvious: the `Public` modifier was hard-coded into the generated code, which caused private fields to turn into `Public` properties; the fix takes it in as a parameter, allowing `Private` fields to turn into `Private` properties.
